### ISSUE TITLE
1.2.3 on server collision

### ISSUE THREAD

**Issue by superon** [Hephaestus-Dev/TinyTweaks#78]
i suspect you may already be aware of this one.

i've set

  "leaves": {
    "collide": false,

on the server but setting this to true on the client allows them to walk on them on the server :/

**Comment by superon** [comment]
also noting that though this appears to be true for natural leaves (those generated by forests etc) it's not the same for those placed by the player.
generated leaves do still have collision whilst placed leaves don't.

this statement is noting behaviour when client has collision true whilst server has it false.

**Comment by Haven-King** [comment]
Yes, unfortunately Minecraft handles collisions client-side. Syncing configs to the client is something I've looked into, but likely won't be added any time soon.
